### ISSUE TITLE
Command 18 "send binlog"

### ISSUE THREAD

**Issue by mcronce** [vitessio/vitess#6973]
#### Feature Description
While running mysql-haskell's set of integration tests against Vitess, one feature they check is the "send binlog" command.  I don't think that the feature is necessarily critical to the operation of that client, but it is a feature that they check.

The specific feature we're looking for is a Vitess-level implementation of MySQL's "send binlog" (18) command.  This is an example of a request payload from the client:
```
0000   23 00 00 00 12 aa 3c 00 00 00 00 ea 03 00 00 76
0010   74 2d 31 36 36 32 38 38 34 38 32 36 2d 62 69 6e
0020   2e 30 30 30 30 30 31
```

I will attach a screenshot of the human-readable values from Wireshark.

For the time being I'm going to leave this particular test out of the test suite for that framework.  Once the feature is implemented, I'll add it back.

**Comment by harshit-gangal** [comment]
I do not think this is something we can support in sharded system. What does this would mean to a user?

cc: @deepthi @systay

**Comment by harshit-gangal** [comment]
This can be used to incorporate Vstream through MySQL protocol.
